simapp/app.go

```diff
@@ -410,6 +410,8 @@ func NewSimApp(
 		appCodec, runtime.NewKVStoreService(keys[ibctransfertypes.StoreKey]), app.GetSubspace(ibctransfertypes.ModuleName),
 		app.IBCFeeKeeper, // ISC4 Wrapper: fee IBC middleware
 		app.IBCKeeper.ChannelKeeper,
+		app.IBCKeeper.ChannelKeeperV2,
```

Comment:
This change does have one important implication: you can't wire up transfer without also creating all the Eureka keepers also. I guess we probably want people to do that, but I think up until now this was not strictly needed, right?

Reply:
You can pass in nil here if you are only supporting classic

Reply:
Nice!